tests/Romania/RomaniaTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ *  This file is part of the Yasumi package.
+ *
+ *  Copyright (c) 2015 - 2017 AzuyaLabs
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <stelgenhof@gmail.com>
+ */
+namespace Yasumi\tests\Romania;
+
+use Yasumi\Holiday;
+
+/**
+ * Class RomaniaTest
+ * @package Yasumi\tests\Romania
+ */
+class RomaniaTest extends RomaniaBaseTestCase
+{
+    /**
+     * @var int year random year number used for all tests in this Test Case
+     */
+    protected $year;
+    /**
+     * Tests if all national holidays in Romania are defined by the provider class
+     */
+    public function testNationalHolidays()
+    {
+        $this->assertDefinedHolidays([
+            'newYearsDay',
+            'dayAfterNewYearsDay',
+            'unitedPrincipalitiesDay',
+            'internationalWorkersDay',
+            'easter',
+            'easterMonday',
+            'childrensDay',
+            'pentecost',
+            'pentecostMonday',
+            'assumptionOfMary',
+            'stAndrewDay',
+            'nationalDay',
+            'christmasDay',
+            'secondChristmasDay',
+        ], self::REGION, $this->year, Holiday::TYPE_NATIONAL);
+    }
+    /**
+     * Tests if all observed holidays in Romania are defined by the provider class
+     */
+    public function testObservedHolidays()
+    {
+        $this->assertDefinedHolidays([
```

Comment:
This test fails for years before 2016, as this holiday didn't exist yet before 2016. I ran multiple unit tests and it came out as a fail:

```
Yasumi\tests\Romania\RomaniaTest::testObservedHolidays
Failed asserting that an array has the key 'constantinBrancusiDay'.
```

Best is to run your unit tests repeatedly, e.g. `vendor/bin/phpunit --testsuite=Romania --repeat=100` to ensure nothing is missed.